libs/react/src/lib/table/story/multiple-story-tables.tsx

```diff
@@ -124,7 +124,7 @@ export const SortableTable = () => {
       <TableHeader>
         <TableRow>
           {columnData.map((column: SuperHeroColumnDataT, i: number) => (
-            <TableHeaderCell key={i} {...column} />
+            <TableHeaderCell key={i} {...column} tooltipText={'Tooltip'} />
```

Comment:
Its better to pass children to `TableHeaderCell`  here:
```jsx
<TableHeaderCell key={i} {...column}>
    <TooltipButton tooltip={tooltipText} className="sg-table-sort">
        {children}
    </TooltipButton>
</TableHeaderCell>
```
and remove the logic from the Table component. That way we keep components seperate and minimize dependencies.

Reply:
Makes sense! I'll get started on that!

Reply:
How is this going @samspi? Can we assist you in any way?

Reply:
Hi there. An update. I've been unable to dedicate time to these fixes and I'm looking into handing it over to another member in the team. Thanks for your patience!